src/components/form-components/SearchInput.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FormikPropType from 'utils/formik-prop-types';
+import { Search } from 'react-feather';
+
+const SearchInput = (props) => {
+  const {
+    label, value, onChange, name, id, placeholder,
+    className, width, size, formik, onBlur, onSubmit,
+  } = props;
+
+  const classList = [];
+  if (className) {
+    classList.push(className);
+  }
+  if (size) {
+    classList.push(`input--${size}`);
+  }
+  classList.push('input border');
+  classList.push('w-full');
+
+  if (formik && formik.touched[name] && formik.errors[name]) {
+    classList.push('error');
+  }
+
+  const endId = id || `id_${name}`;
+
+  return (
+    <div className={`${width} mb-3 flex items-center justify-left relative inline-block`}>
+      {label && (
+        <label htmlFor={endId}>{label}</label>
+      )}
```

Comment:
Is SearchInput need to have label?

Reply:
I would consider the option of adding the searchBox=true/false property to TextInput